pkg/resourcemanager/azuresql/azuresqluser/azuresqluser.go

```diff
@@ -203,17 +197,14 @@ func (m *AzureSqlUserManager) DeleteSecrets(ctx context.Context, instance *v1alp
 			"password",
 		}
 
-		for _, formatName := range customFormatNames {
-			key := types.NamespacedName{Namespace: secretKey.Namespace, Name: instance.Name + "-" + formatName}
-
-			err = secretClient.Delete(
-				ctx,
-				key,
-			)
-			if err != nil {
-				instance.Status.Message = "failed to delete secret, err: " + err.Error()
-				return false, err
-			}
+		err = secretClient.Delete(
```

Comment:
This might be a problem because of the permissions that RBC has set up?

Might need to move this to best effort for now?

Reply:
Nope it's ok we were doing this before too.